### PR TITLE
fix: Remove FQDN and $edit

### PR DESCRIPTION
**Pull request by nschonni**

### CODE REVIEW

File: files/fr/mozilla/add-ons/webextensions/add_a_button_to_the_toolbar/index.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ Dans WebExtensions, ces types de boutons s’appellent « actions du navigateur
 
 ## Un bouton simple
 
-Dans cette section, nous créerons une WebExtension qui ajoute un bouton à la barre d’outils. Lorsque l’utilisateur clique sur le bouton, nous ouvrirons [https&nbsp;://developer.mozilla.org](https://developer.mozilla.org) dans un nouveau onglet.
+Dans cette section, nous créerons une WebExtension qui ajoute un bouton à la barre d’outils. Lorsque l’utilisateur clique sur le bouton, nous ouvrirons [https&nbsp;://developer.mozilla.org](/) dans un nouveau onglet.
```
Comment (cw118) [inline files/fr/mozilla/add-ons/webextensions/add_a_button_to_the_toolbar/index.md:24]:
```suggestion
Dans cette section, nous créerons une WebExtension qui ajoute un bouton à la barre d’outils. Lorsque l’utilisateur clique sur le bouton, nous ouvrirons [https://developer.mozilla.org](/) dans un nouveau onglet.
```

File: files/fr/web/javascript/reference/classes/public_class_fields/index.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ original_slug: Web/JavaScript/Reference/Classes/Class_fields
 >
 > Les déclarations de champs, tant publics que privés, sont [une fonctionnalité expérimentale (étape 3)](https://github.com/tc39/proposal-class-fields) proposée au [TC39](https://tc39.github.io/beta/), le comité des standards JavaScript.
 >
-> La prise en charge dans les navigateurs est limitée, mais cette fonctionnalité peut être utilisée à travers une étape de contruction avec des systèmes tels que [Babel](https://babeljs.io/). Voir [l'information de compatibilité](/fr/docs/Web/JavaScript/Reference/Classes/Class_fields$edit#Browser_compatibility) ci-dessous.
+> La prise en charge dans les navigateurs est limitée, mais cette fonctionnalité peut être utilisée à travers une étape de contruction avec des systèmes tels que [Babel](https://babeljs.io/). Voir [l'information de compatibilité](/fr/docs/Web/JavaScript/Reference/Classes/Class_fields#Browser_compatibility) ci-dessous.
```
Comment (cw118) [inline files/fr/web/javascript/reference/classes/public_class_fields/index.md:18]:
```suggestion
> La prise en charge dans les navigateurs est limitée, mais cette fonctionnalité peut être utilisée à travers une étape de contruction avec des systèmes tels que [Babel](https://babeljs.io/). Voir [l'information de compatibilité](/fr/docs/Web/JavaScript/Reference/Classes/Public_class_fields#compatibilité_des_navigateurs) ci-dessous.
```

**Review by cw118 — CHANGES_REQUESTED** [review]
Two somewhat unrelated comments, but since there are not too many changes here I thought these could be corrected in this PR as well :)

File: files/fr/web/javascript/reference/classes/public_class_fields/index.md (hunk 2 above)
Comment (nschonni) [reply files/fr/web/javascript/reference/classes/public_class_fields/index.md:18]:
```suggestion
> La prise en charge dans les navigateurs est limitée, mais cette fonctionnalité peut être utilisée à travers une étape de contruction avec des systèmes tels que [Babel](https://babeljs.io/). Voir [l'information de compatibilité](#compatibilité_des_navigateurs) ci-dessous.
```

**Review by cw118 — APPROVED** [review]
Thank you for fixing the link/anchor in my suggestion and thanks so much for the PR !

**Pull request merged**